analyzer/consensus/consensus.go

```diff
@@ -907,6 +908,19 @@ func (m *processor) queueSubmissions(batch *storage.QueryBatch, data *governance
 				submission.CreatedAt,
 				submission.ClosesAt,
 			)
+		case submission.Content.ChangeParameters != nil:
+			batch.Queue(queries.ConsensusProposalSubmissionChangeParametersInsert,
+				submission.ID,
+				submission.Submitter.String(),
+				submission.State.String(),
+				submission.Deposit.String(),
+				submission.Content.ChangeParameters.Module,
+				submission.Content.ChangeParameters.Changes,
```

Comment:
Changes is `cbor.RawMessage` of one of the various [`ConsensusParameterChanges`](https://github.com/search?q=repo%3Aoasisprotocol%2Foasis-core+ConsensusParameterChanges&type=code) types.

We could probably try unmarshaling this into `var changes map[string]interface{}` and then marshaling it into JSON, and storing a JSONB. I think that should work in all cases that we are likely to ever hit, but not absolutely certain if there are any potential edge cases, so for now opted to just storing the raw value. The frontend or the API controller could still do this.